typings.d.ts

```diff
@@ -179,11 +205,17 @@ declare module 'resin-components' {
 		class h4 extends RenderableElementWithProps<FlexProps, any> {}
 	}
 
-	interface LinkProps
-		extends DefaultProps,
-			React.AnchorHTMLAttributes<HTMLElement> {
+	interface LinkProps extends DefaultProps {
 		blank?: boolean;
 		disabled?: boolean;
+		download?: any;
+		href?: string;
+		hrefLang?: string;
+		media?: string;
+		rel?: string;
+		target?: string;
+		type?: string;
+		as?: string;
```

Comment:
Are `type` and `as` necessary? Aren't those for `<link>` tags?
Couldn't find a place that these are used in the source.

Reply:
These are taken from the React typings

Reply:
Couldn't find those two to be part of `AnchorHTMLAttributes` and that's why I mentioned it.